requirements-dev.in

```diff
@@ -1,10 +1,12 @@
--r requirements.txt
-pytest
-pytest-cov
-coverage
+-c requirements.txt
+
 black
+boto3-stubs[cognito-idp,dynamodb,events,lambda,s3,sns,ssm,sqs]<1.22.0
+coverage
 flake8
-pylint<2.11
```

Comment:
_FYI:_ Most likely this `<2.11` version was the problem as I got this error when I tried to freeze dependencies for the first time:

![image](https://user-images.githubusercontent.com/15995354/182021858-1bafe332-8bb8-4521-97f3-79ad97cf02f6.png)